packages/core/src/RenderingEngine/RenderingEngine.ts

```diff
@@ -1240,6 +1220,7 @@ class RenderingEngine implements IRenderingEngine {
 
     const onScreenContext = canvas.getContext('2d');
 
+    const rect = canvas.getBoundingClientRect();
```

Comment:
is there any value in calling getBoundingClientRect here? I don't see it being used